api/jreleaser-model-api/src/main/java/org/jreleaser/version/CalVer.java

```diff
@@ -465,7 +465,7 @@ public static CalVer defaultOf(String format) {
 
         return of(format, format.replace(T_YEAR_LONG, "2000")
             .replace(T_YEAR_SHORT, "0")
-            .replace(T_YEAR_ZERO, "0")
+            .replace(T_YEAR_ZERO, "00")
```

Comment:
This will make the year `2000` be equal to `00` for both `0Y` and `YY`, right?

Reply:
For `YY`, value `00` is invalid.

I added a test in `version_invalid` for that:
![obraz](https://github.com/user-attachments/assets/23d27a9c-5254-43e4-8616-6c05dc0f2f30)

Reply:
I guess the year `2000` is special and should match both `0Y` and `YY` making `00` valid for both formats.

Reply:
If so, I can update the PR 🙂

Reply:
But I wonder, doesn't this look better if someone use `YY` format, to version like:
- `0`
- `1`
- `2`
- `3`
- (...)
- `24`

instead of:
- `00`
- `1`
- `2`
- `3`
- (...)
- `24`

?

And with `0Y` it would be:
- `00`
- `01`
- `02`
- `03`
- (...)
- `24`

Reply:
True. Let's keep `0` for `YY`. In which case, the PR needs no updates, right?

Reply:
Yes, correct. No update needed in this case